specs/Freeze.md

```diff
@@ -0,0 +1,155 @@
+# Background
+Edge browser has a sleeping tab feature which reduces resource usage when the tab is in the background. We are introducing WebView2 APIs
+to access this feature so that invisible WebView can use less resources. We'd appreciate your feedback.
+
+
+# Description
+You may call the `TryFeezeAsync` API to have the WebView2 consume less memory. This is useful when your Win32 app becomes invisible, or when your Universal Windows Platform app is suspended, during the suspended event handler before completing the suspended event.
+
+# Examples
+## .Net, WinRT
+```c#
+async protected void OnSuspending(object sender, SuspendingEventArgs args)
+{
+    SuspendingDeferral deferral = args.SuspendingOperation.GetDeferral();
+    // Set webView.Visibility to false first.
+    // WebView2 must be invisible for TryFreezeAsync to succeed.
+    webView.Visibility = false;
```

Comment:
Is it necessary to collapse as part of Suspending? Isn't the whole Window becoming invisible anyway? Or does this trigger the WV2 to go to the CWV2Controller and set IsVisible to false? If so, not very obvious, worth calling out.

Reply:
Updated the comment. It is CWV2Controller's visibility that matters